### PR TITLE
Drop Node v6 version support

### PR DESCRIPTION
**Pull request by SparshithNR**
1. Updating the package.json
2. Update the CI tests
3. Adding Node v12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +62,6 @@
     "sinon-chai": "^3.1.0"
   },
   "engines": {
-    "node": "6.* || >= 8.*"
+    "node": "8.* || >= 10.*"
```
Comment (rwjblue) [inline package.json]: We do not want to support Node 11 (it is unsupported by Node itself at this point):

```suggestion
    "node": "8.* || 10.* || >= 12.*"
```

Comment (SparshithNR) [reply package.json]: Should I add the node 12 to the CI as well?

Comment (rwjblue) [reply package.json]: Yeah, probably

**Comment by rwjblue** [comment]
OK, lets get the other PR's ready before we land them all and release the major bump.

**Comment by SparshithNR** [comment]
@stefanpenner can we merge this change to master?

**Pull request merged**